aframe/src/location-based/gps-entity-place.js

```diff
@@ -80,7 +84,10 @@ AFRAME.registerComponent('gps-entity-place', {
 
         position.z = this._cameraGps.computeDistanceMeters(this._cameraGps.originCoords, dstCoords, true);
         position.z *= this.data.latitude > this._cameraGps.originCoords.latitude ? -1 : 1;
-
+        if(this.data.altitude !== 0) {
+            position.y = this.data.altitude - this._cameraGps.originCoords.altitude;
```

Comment:
I would like to leave this 'altitude' for the object.

I would like to use the `position.y`  if specified, or 0. That's because what users usually want to do is to change height of object, not altitude (the height from the sea level). So, we already have position.y (aframe attribute) that, if specified, is used.

Saying that, I'm ok with simulating altitude as a global altitude, on the `gps-camera` as you did.

So the line above will become:

` position.y = position.y - this._cameraGps.originCoords.altitude;`